test/testconcaturl.cpp

```diff
@@ -14,7 +14,7 @@
 
 using namespace OCC;
 
-typedef QList< QPair<QString,QString> > QueryItems;
+using QueryItems = QList< QPair<QString,QString> >;
```

Comment:
While at it you can remove the pre-C++11-style spaces after/before `<`/`>`, respectively.

Reply:
Fixed one such case but I missed that one it seems. Thanks.